unittest/meta/TestSaiSerialize.cpp

```diff
@@ -1063,6 +1063,7 @@ TEST(SaiSerialize, serialize_qos_map)
     EXPECT_EQ(l.key.queue_index, 6);
     EXPECT_EQ(l.key.color, SAI_PACKET_COLOR_RED);
     EXPECT_EQ(l.key.mpls_exp, 0);
+    EXPECT_EQ(l.key.fc, 0);
```

Comment:
you could actually specify some non zero value for better check

Reply:
Done